3rdPartyCypress/cypress/integration/errorPageNews.js

```diff
@@ -0,0 +1,32 @@
+import {
```

Comment:
There is also an `errorPagePersian` file that will test the Mozart pages.

Renaming this directory so it is namespaced under `cypressThirdParty` would keep the code in a similar place in the directory structure. Easier for onboarding. 


There should be a brief README.md in this directory explaining what should go here as opposed to the `cypress` directory. 

We also have integration tests to check manifest and serviceworker files are being served. Since these rely on Sling to upload the files, they could potentially be classed as a third party dependency.

Reply:
I deliberately didn't think them being next to each other was good as it being further from /cypress means one is less likely to click the wrong folder based on looking for the word cypress.

re making it clearer I will add documentation later and also not reuse the helpers (as this can cause errors only on latest). But I want this merged ASAP just to unblock deploy so kept my changes narrow here.

Reply:
Forgot to respond to "There is also an errorPagePersian file that will test the Mozart pages." There is but it's not live yet, so I cut it. There's no need to test it on test.bbc.com IMO.

Reply:
Okay. So long as the documentation is added shortly - including why the error page work is split across the `cypress` and `3rdPartyCypress` directories, the differences between the services that are live and those that are not, I'm happy for this to be merged.